server/embed/serve.go

```diff
@@ -316,6 +328,7 @@ func (sctx *serveCtx) createMux(gwmux *gw.ServeMux, handler http.Handler) *http.
 					},
 				),
 				wsproxy.WithMaxRespBodyBufferSize(0x7fffffff),
+				wsproxy.WithLogger(wsProxyZapLogger{sctx.lg}),
```

Comment:
Could you please provide examples how to validate the fix works as expected? Thanks!

Reply:
I will try to check

Reply:
@chaochn47 it works!

```
мар 11 01:03:34 localhost etcd[7550]: {"level":"warn","ts":"2023-03-11T01:03:34.272+0500","caller":"embed/serve.go:287","msg":"error reading websocket message:websocket: close 1006 (abnormal closure): unexpected EOF"}
```

You need to establish correct websocket connection to etcd. And kill -9 the websocket client. It will show the message above.

Please merge!